examples/multi-gpu-movielens/torch_trainer_dist.py

```diff
@@ -0,0 +1,149 @@
+import argparse
+import glob
+import os
+from time import time
+
+import cupy
+import torch
+import torch.distributed as dist
+import torch.nn as nn
+
+import nvtabular as nvt
+from nvtabular.framework_utils.torch.models import Model
+from nvtabular.framework_utils.torch.utils import process_epoch
+from nvtabular.loader.torch import DLDataLoader, TorchAsyncItr
+
+parser = argparse.ArgumentParser(description="Train a multi-gpu model with Torch and Horovod")
```

Comment:
remove `Horovod` from the description and add `Torch DistributedDataParallel` may be?